seatsio-android-component/src/main/java/io/seats/seatingChart/SectionCategory.java

```diff
@@ -0,0 +1,41 @@
+package io.seats.seatingChart;
+
+import com.google.gson.annotations.Expose;
+
+import java.util.NoSuchElementException;
+
+import io.seats.utils.Either;
+
+public class SectionCategory {
```

Comment:
What's the reason for having a separate `SectionCategory` class? Doesn't a section just have a regular `Category`?